feature/auth/src/main/java/com/ku_stacks/ku_ring/auth/compose/component/AuthDialog.kt

```diff
@@ -0,0 +1,99 @@
+package com.ku_stacks.ku_ring.auth.compose.component
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.PaddingValues
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.style.LineHeightStyle
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import androidx.compose.ui.window.Dialog
+import androidx.compose.ui.window.DialogProperties
+import com.ku_stacks.ku_ring.auth.compose.component.button.RoundedCornerButton
+import com.ku_stacks.ku_ring.designsystem.components.LightAndDarkPreview
+import com.ku_stacks.ku_ring.designsystem.kuringtheme.KuringTheme
+import com.ku_stacks.ku_ring.designsystem.kuringtheme.values.Pretendard
+import feature.auth.R.string.dialog_heading_sign_in_wrong
+import feature.auth.R.string.dialog_sub_heading_sign_in_wrong
+import feature.auth.R.string.dialog_proceed_sign_in_wrong
+
+@Composable
+fun AuthDialog(
```

Comment:
```suggestion
internal fun AuthDialog(
```

- 외부에서 접근할 수 없도록..?
- Modifier도 주입안되어있는...그냥 한두번 사용하는 컴포넌트인 것 같은데

Reply:
수정: https://github.com/ku-ring/KU-Ring-Android/pull/437/commits/243eabedd09131f2aded040f43111bbab99dbc14

이 부분은 컴포넌트 패키지에 두는 대신 signin 패키지 내부로 이동시켰습니다!